config/routes.rb

```diff
@@ -3,13 +3,14 @@
   devise_for :users
 
   resources :parks, only: [:index, :show, :new, :create, :edit, :update, :destroy] do
-    resources :reviews, only: [:index, :new, :create, :edit]
+    resources :reviews, only: [:index, :new, :show, :create, :destroy]
   end
+  resources :reviews
 
   namespace :api do
     namespace :v1 do
       resources :parks, only: [:index, :show, :new, :create, :update, :edit] do
-        resources :reviews, only: [:index,:show, :new, :create]
+        resources :reviews, only: [:index, :show, :new, :create, :update, :edit, :destroy]
```

Comment:
Looks like you have ALL 7 actions now so no need to use only anymore, please remove everything except resources :reviews